<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>451</width>
      <height>501</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>217</width>
      <height>0</height>
     </size>
    </property>
    <property name="title">
     <string>영상</string>
    </property>
    <widget class="QPushButton" name="btu_finish">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>191</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Finish</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btu_start">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>201</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="tex_folderName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>460</y>
       <width>341</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btu_picture">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>460</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>등록하기</string>
     </property>
    </widget>
    <widget class="QListWidget" name="list_att">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>431</width>
       <height>401</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>231</width>
      <height>501</height>
     </rect>
    </property>
    <property name="title">
     <string>로그</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="0">
      <widget class="QPushButton" name="btu_clear">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QListWidget" name="list_log"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>705</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="tap_pictrue_2">
   <property name="text">
    <string>tap_pictrue</string>
   </property>
  </action>
  <action name="tap_pictrue">
   <property name="text">
    <string>kkkk</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
